keyboards/cannonkeys/satisfaction75/prototype/info.json

```diff
@@ -1,24 +1,34 @@
 {
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B9", "pin_b": "B8", "resolution": 2}
+        ]
+    },
+    "matrix_pins": {
+        "cols": ["B1", "B2", "B10", "B11", "B12", "B13", "B14", "A8", "A9", "A10", "B0", "A7", "A5", "B5", "A15", "A1"],
+        "rows": ["B3", "B4", "A0", "A2", "A4", "A3"]
+    },
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x57F5"
+    },
     "layouts": {
         "LAYOUT_all": {
             "layout": [
                 {"label": "K000", "matrix": [0, 0], "x": 0, "y": 0},
-
```

Comment:
Please keep the spacing here.

Reply:
The info.json auto-formatter removes these - I think that should be the standard.